src/main/webapp/app/assessment/assessment-shared.module.ts

```diff
@@ -12,7 +12,6 @@ import { ExternalSubmissionButtonComponent } from 'app/exercises/shared/external
 import { AssessmentWarningComponent } from 'app/assessment/assessment-warning/assessment-warning.component';
 import { ArtemisComplaintsForTutorModule } from 'app/complaints/complaints-for-tutor/complaints-for-tutor.module';
 import { AssessmentLocksComponent } from 'app/assessment/assessment-locks/assessment-locks.component';
-import { MomentModule } from 'ngx-moment';
```

Comment:
I assume we don't need to add here a datejs module or something like that to replace it?